### DISCUSSION TITLE
- MSFT: No data found for this date range, symbol may be delisted

### DISCUSSION THREAD

**Discussion by Tony-Akani** [ranaroussi/yfinance#1910] [Q&A]
Hello I have not used yfinance for a while, so maybe this might sound like an old question. I am trying to fetch OHLCV data for a custom ticker using a custom date, and interval. below is my code

# available intervals = [1m,2m,5m,15m,30m,60m,90m,1h,1d,5d,1wk,1mo,3mo]
symbol = "MSFT"
start_time = "2024-04-01"
end_time = "2024-04-10"
data = yf.download(tickers=symbol, start=start_time, end=end_time, interval="1h")

any idea why i am getting the error 

[*********************100%***********************]  1 of 1 completed

1 Failed download:
- MSFT: No data found for this date range, symbol may be delisted

**Comment by ValueRaider** [comment]
Use the "new issue" form.
